### PR TITLE
remove _json_from_json patch

### PR DESCRIPTION
**Pull request by colin-rogers-dbt**
remove _json_from_json patch

addresses: #1055 

[docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose) dbt-labs/docs.getdbt.com/#

### Checklist

- [ ] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [ ] I have run this code in development and it appears to resolve the stated issue
- [ ] This PR includes tests, or tests are not required/relevant for this PR
- [ ] This PR has no interface changes (e.g. macros, cli, logs, json artifacts, config files, adapter interface, etc) or this PR has already received feedback and approval from Product or DX

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # TODO: how to automate switching from develop to version branches?
 git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
 git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
-
+git+https://github.com/googleapis/python-bigquery.git#egg=google-cloud-bigquery
```
Comment (colin-rogers-dbt) [inline dev-requirements.txt:5]: will remove before merging (but after bigquery has released the next patch)

**Comment by mikealfare** [comment]
@colin-rogers-dbt I converted this to a draft while we wait for the fix to go out in bigquery.

**Review by mikealfare — APPROVED** [review]

**Comment by mikealfare** [comment]
@colin-rogers-dbt Does this need to be backported? If so, please add it to the release ticket (and apply the backport labels of course).

**Comment by dbeatty10** [comment]
This looks like it will fix https://github.com/dbt-labs/dbt-bigquery/issues/1101 also.

We'd just want to add the following test case also:

```shell
dbt show --inline 'select parse_json(null)'
```

**Pull request merged**